php/psalm.xml

```diff
@@ -2,6 +2,8 @@
 <psalm
     errorLevel="1"
     resolveFromConfigFile="true"
+    findUnusedBaselineEntry="true"
+    findUnusedCode="true"
```

Comment:
I'd put these on to false. To get some warnings out of the way and then tackle what is left piece meal. Especially in the generated code it will be hard to follow psalms recommendations.

Reply:
Weird. I can't reproduce that locally. Pushed a commit to see if it matters.